notebooks/official/pytorch/pytorch-text-sentiment-classification-custom-train-deploy.ipynb

```diff
@@ -0,0 +1,1979 @@
+{
```

Comment:
<p>Run Custom Job on Vertex AI Training with a pre-built container&nbsp;-&gt; Run a custom job in Vertex AI using a pre-built container</p><p> </p><p>Hugging Face Transformers library -&gt; the Hugging Face Transformers library</p><p> </p><p>sentiment analysis task -&gt; sentiment analysis tasks</p><p> </p><p>as -&gt; because</p><p> </p><p>that can used -&gt; that you can use</p><p> </p><p>Machine Learning framework -&gt; machine learning framework</p><p> </p><p>Vertex AI Training -&gt; Vertex AI </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>